### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Azure Device Log In_ Keyboard: Keyboard focus is not going to the link present inside “Azure Device Sign In” pop up.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3326]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** MAS2.1.1

**Repro Steps:** 
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application.
7)Tab till “Azure Sign In” button and press enter.
8)Tab till “Sign In” button and press enter.
9)Verify focus is going to the link present inside pop up.

**Actual Result:**
Keyboard focus is not going to the link present inside “Azure Device Sign In” pop up.

**Expected Result:**
Keyboard focus should go to the link present inside “Azure Device Sign In” pop up.

**User Impact:**
Keyboard users can not perform all operations if few controls are not accessible using keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
[Bug_3326_Keyboard issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451211/Bug_3326_Keyboard.issue.pptx)

**Comment by Flanker32** [comment]
We use HyperlinkLabel in IntelliJ to show hyperlink in JLabel, which is not focusable with `tab`. Toolkit will open the browser and open the correspond website when click the link, however, users could tab to button `copy&open` and press enter, which will also open the browser and copy the device code. So it will not block keyboard users.

**Comment by Flanker32** [comment]
- Importance of experience: High, users need to open the auth page while device login.
- Quality of experience: not impacted. Users could open the auth page by click button `Copy&Open`, which is more common used.
- Alternate solutions: We could use `LinkLabel` which could be focused by tab, however, `LinkLabel` doesn't support text before link and text after link, it could show the hyperlink only.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Accessibility56 Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749782
